test/count.test.js

```diff
@@ -1,19 +1,18 @@
 /* eslint-env mocha */
-import * as assert from 'proclaim';
+/* global proclaim sinon */
 import * as fixtures from './helpers/fixtures';
 import fetchMock from 'fetch-mock';
-import sinon from 'sinon/pkg/sinon';
 import Count from '../src/js/count';
 
-sinon.assert.expose(assert, {
+sinon.proclaim.expose(proclaim, {
```

Comment:
```suggestion
sinon.assert.expose(proclaim, {
```

https://sinonjs.org/releases/latest/assertions/#sinonassertexposeobject-options